client/src/AddCommentForm.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { PropTypes } from 'react';
+
+class AddCommentForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      author: '',
+      text: '',
+    };
+  }
+
+  handleAuthorChange = (evt) => {
+    this.setState({ author: evt.target.value });
+  }
+
+  handleTextChange = (evt) => {
+    this.setState({ text: evt.target.value });
+  }
+
+  handleSubmit = (evt) => {
+    evt.preventDefault();
+    const author = this.state.author.trim();
+    const text = this.state.text.trim();
+    const poemId = this.props.poemId;
```

Comment:
All usage of poem ID in this method -- and indeed in this class -- is now unnecessary. See comments below.